modules/ghservice/manifests/deps/nodejs.pp

```diff
@@ -1,11 +1,22 @@
-class ghservice::deps::nodejs () {
-
-    # Get the version from hiera
-    $nodejs_version = hiera('nodejs_version', '0.10.35-1nodesource1~trusty1')
+class ghservice::deps::nodejs (
+    $nodejs_version = "0.10.35-1nodesource1~trusty"
+    ) {
 
     # Apply the nodejs class which will configure the apt repo and install nodejs
     class { '::nodejs':
         version => $nodejs_version,
         manage_repo => true
     }
+
+    # This next section is necessary to ensure the above class
+    # is actually contained within the surrounding class
+    # because by default puppet will let that nodejs declaration float freely in
+    # the dependency graph, resulting in commands like npm install trying
+    # to run too early.
+    # https://docs.puppetlabs.com/puppet/latest/reference/lang_containment.html#anchor-pattern-containment-for-compatibility-with-puppet--340
```

Comment:
As a note in general on this PR, can you clean up this documentation a bit so that all lines have equal length? Makes it all more readable